installer/PowerToysSetup/Product.wxs

```diff
@@ -223,6 +223,7 @@
               <?undef IdSafeLanguage?>
             <?endforeach?>
             <Directory Id="FileExplorerPreviewInstallFolder" Name="FileExplorerPreview" />
+            <Directory Id="FancyZonesInstallFolder" Name="FancyZones" />
```

Comment:
Name is quite misleading as it's FancyZones and fz is in another directory.
How about FancyZonesEditor?

Reply:
We will move all the FancyZones binaries, to that folder, right now we need it for the editor, but in 0.19 we will move the remaining parts.